examples/har_extractor.py

```diff
@@ -239,6 +216,18 @@ def done(context):
     )
 
 
+def format_cookies(obj):
+    if obj:
+        return [{"name": k.strip(), "value": v[0]} for k, v in obj.items()]
+    return ""
+
+
+def format_headers(obj):
+    if obj:
+        return [{"name": k, "value": v} for k, v in obj.fields]
+    return ""
+
+
 def print_attributes(obj, filter_string=None, hide_privates=False):
```

Comment:
Um, I could come with an example, but it'll probably be something obtuse since I've never actually used this function. I guess it was added by the original author to quickly debug some objects.